src/pages/DatasetStatistics.tsx

```diff
@@ -66,6 +83,31 @@ export default function DatasetStatistics(props: DatasetStatisticsProps) {
     const init = async () => {
       try {
         const dataset: Dataset = await DataSet.getDatasetByDatasetIdentifier(datasetIdentifier);
+        const datasetTerms: DatasetTerm[] = await DataSet.searchDatasetIndex({ query: {
```

Comment:
I might be missing something, but it seems like in order to populate all of the information on this page, we unfortunately need both the Dataset _and_ the DatasetTerms

Reply:
If you're still missing some required details, let's discuss. We can easily add new values to the `DatasetTerm` object that is indexed in ES.